app/controllers/carl_people_controller.rb

```diff
@@ -0,0 +1,47 @@
+class CarlPeopleController < ApplicationController
+  #----------------- SHOW A PERSON --------------------
+  def show
+    @person = Person.find(params[:id])
+    @tasks_count = @person.tasks.count
+    @tasks_wip = @person.tasks.where(date_complete: nil).count
+    @tasks_done = @tasks_count - @tasks_wip
+  rescue
```

Comment:
In general I would avoid blanket rescues like this - it's an easy way to hide bugs unrelated to what you wanted to rescue.